src/locales/en/translation.json

```diff
@@ -30,6 +30,7 @@
   "contactUs": "Contact us",
   "contacts": "Contacts",
   "continueWith": "Continue with",
+  "copyrights": "©2020 – Data Ocean. All rights reserved.",
```

Comment:
```suggestion
  "allRightsReserved": "All rights reserved",
```

Reply:
"©2020 – Data Ocean"  -  should i leave it on page?

Reply:
yes